src/data/faq.json

```diff
@@ -491,16 +491,30 @@
+                            "To resolve the problem, please ensure that you are using the latest version of the operating system available for your smartphone. If this doesn't help, please contact your device manufacturer.",
+                            "<b>Both Check-in and Contact Journal functionality, as well as retrieving PCR and rapid test results, is fully functional without restriction</b> and only proximity-based ID recording is affected.",
+                            "<hr/>",
+                            "Technical explanation: Your hardware is capable of Bluetooth Low Energy, but your Bluetooth driver is missing a capability called “peripheral mode”. This can only be fixed by your device manufacturer through an update of your operating system. Because the capability is missing, the Corona-Warn-App cannot cause your device to periodically broadcast its own random IDs via Bluetooth."
+                        ]
+                    },
+                    {
+                        "title": "[Google/Android] A card on my Start Screen is notifying me that my device is incompatible. What does this mean?",
+                        "anchor": "full_incompat",
+                        "active": true,
+                        "textblock": [
+                            "This does not usually happen on unmodified end-user devices. Therefore, this section doesn't apply to the vast majority of devices. Usually, your device would only be <a href='#part_incompat' target='_blank' rel='noopener'>partially incompatible</a>.",
+                            "In version 2.2, a check regarding Bluetooth capabilities of your device was added to the Corona-Warn-App for Android. If a warning message that links to this FAQ entry is shown, your device is not capable of Bluetooth Low Energy and can therefore neither send nor receive random IDs.",
+                            "Your risk assesment will only be based on events and locations you were checked in to.",
+                            "Possible reasons for this message are:<ul><li>Your device does not contain the hardware that is required to use Bluetooth Low Energy.</li><li>Your device does not contain the software required to use the Bluetooth Low Energy capabilities of your hardware.</li><li>You are running the device in the Android Emulator.</li></ul>",
+                            "When in doubt, please ensure that you are using the latest version of the operating system available for your smartphone. For further assistance, please contact your device manufacturer.",
+                            "<b>Both Check-in and Contact Journal functionality, as well as retrieving PCR and rapid test results, is fully functional without restriction</b> and only proximity-based ID recording is affected."

```

Comment:
Same here reg. "is" -> "are"

Reply:
Suggest also here leaving out this problematic sentence. The message should be covered by L514.